README.md

```diff
@@ -5,7 +5,11 @@ Adapted from https://github.com/hpcloud-mon/ansible-percona
 
 ## Requirements
 
-None.
+Ansible >= 2.5
+
+For Mysql >= 8.0, the only authentication plugin currently supported is:
+
+`mysql_default_authentication_plugin: "mysql_native_password"`
```

Comment:
I'd remove this lines. It's setting the auth. plugin to whatever is set in the related var and, if the var is not set, it will use `caching_sha2_password` by default.